src/index.js

```diff
@@ -1,16 +1,34 @@
 import angular from 'angular';
 
-import { HelloController } from './app/hello';
-import { Example } from './app/services/example';
+import { SignUpCtrl } from './app/sign_up/sign_up';
+import { SignInCtrl } from './app/sign_in/sign_in';
+import { MapHomeCtrl } from './app/map_home/map_home'
+import { UserService } from './app/services/user_service';
+import { localStorageServiceProvider } from '../node_modules/angular-local-storage/dist/angular-local-storage.js';
+import { angularValidationMatch } from '../node_modules/angular-validation-match/dist/angular-validation-match.js';
+import { AuthInterceptor } from './app/config/auth.interceptor.js';
+import { OverwriteEmail } from './app/directives/overwrite_email.js'
 import 'angular-ui-router';
 import routesConfig from './routes';
-
 import './index.scss';
 
 export const app = 'app';
 
 angular
-  .module(app, ['ui.router'])
+  .module(app, ['ui.router', 'LocalStorageModule', 'validation.match'])
   .config(routesConfig)
-  .controller('hello', HelloController)
-  .service('Example', Example);
+  .controller('SignUpCtrl', SignUpCtrl)
```

Comment:
You can decouple this js file in the following way, which I think will make things more clear for the development:

You can use the controller files and define them like this:
```
(function () {
  'use strict';

  angular
    .module('skillzzWeb')
    .controller('signUpController', signUpController);

    function signUpController(...) {
      ...
    };
})();
```


Same with directives, config and constants.